script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_5.txt

```diff
@@ -55,29 +55,29 @@ She just says something completely outrageous―――!
 We're flying.
 No, we're jumping.
 <Page17>
-Arcueid jumps towards the building wall, bouncing off it after a bit and repeating the motion again in the opposite direction. Before I realize it, we're above the building, no, soaring across the sky.
-Of course, she did all of this while grabbing me by her side.
+Arcueid bounds from wall to wall up the adjoining buildings. Before I realize it, we're above the buildings, no, soaring across the sky.
+Of course, she did all of this while holding onto me.
 <Page18>
 "W-W-Wait a, waaaaaah...!"
-It's so high! I'm terrified! How many meters above ground is this? That building must have been around 40 meters and we're above that!? Even a 20-meter drop-tower ride would make my heart shrink, let alone this! Not to mention our complete lack of safety gear!
+It's so high! I'm terrified! How many meters above ground is this? That building must have been around 40 meters and we're above that!? Even a 20-meter drop-tower ride would make my heart shrink, let alone this! Where the hell am I supposed to put my feet?!
 <Page19>
 "Hmm? Shiki, you'll bite your tongue off if you open your mouth like that."
 <Page20>
 As I tilt my head to cover my screaming, Arcueid ramps up the brutal pace even more.
 We hop off of each rooftop from building to building. Just like a rabbit hopping on the moon.
 <Page21>
 "Uoh, Uooh, Uoooooooooh―――!?"
-My face tightens because of this unfamiliar experience.
+I endure this surreal experience with gritted teeth.//to be rechecked
 Just as I was starting to enjoy it, at long last, Arcueid stops leaping.
 <Page22>
 It's the rooftop of a skyscraper overlooking the city.
 We have landed on an aerial garden that was not originally designed for humans to enter.
 <Page23>
 "Eh―――you, w-what the hell was that about―――"
 <Page24>
-The onslaught of up and down movements disrupted my breathing.
-My chest is in pain. The disordered blood circulation created a strain on my heart.
-Although there's no change in the air pressure, my entire body becomes rigid from the sudden experience that I never had before.
+This rollercoaster experience has played havoc with my breathing.
```

Comment:
"wreaked havoc on" is a much more common expression

Reply:
It is, but it also feels a bit more destructive. Play havoc isn't _unused_ at least 
https://books.google.com/ngrams/graph?content=play+havoc%2C+wreak+havoc&year_start=1800&year_end=2019&corpus=26&smoothing=3&direct_url=t1%3B%2Cplay%20havoc%3B%2Cc0%3B.t1%3B%2Cwreak%20havoc%3B%2Cc0